x11-drivers/nvidia-drivers/nvidia-drivers-460.39.ebuild

```
@@ -223,7 +186,7 @@ donvidia() {
 	# Get just the library name
 	nv_LIBNAME=$(basename "${nv_LIB}")
 
-	if [[ "${nv_DEST}" ]]; then
+	if [[ ${nv_DEST} ]]; then
```

```suggestion
	if [[ -n ${nv_DEST} ]]; then
```

I personally prefer to use -n/-z when it's really about null/non-null value vs pseudo-booleans.